oas

```diff
@@ -840,44 +831,74 @@
+          ]
+      },
+      "Daily": {
+        "type": "object",
+        "properties": {
+         "earliest": {
             "type": "string",
             "format": "date",
-            "description": "Date that the irrigation should performed"
+            "description": "The first date of the time-window that an irrigation application should be applied."
           },
-          "applications": {
-            "type": "array",
-            "items": {
-              "type": "object",
-              "properties": {
-                "date_time": {
-                  "type": "string",
-                  "format": "date-time",
-                  "description": "Date-time that the application of water should performed. The date MUST be the same as the recommendation date"
-                },
-                "ratio": {
-                  "type": "integer",
-                  "description": "Percentage of the daily irrigation. Either all applications have a defined ratio, in which case it must total 100, or none have ratio defined in which case the distribution is assumed to be even."
-                }
-              },
-              "required": [
-                "date_time",
-                "ratio"
-              ]
-            }
+          "latest": {
+            "type": "string",
+            "format": "date",
+            "description": "the last date of the time-window that an irrigation application should be applied."
+          },
+         "applications": {
+            "type": "integer",
+            "description": "An optional parameter that refers to the number of applications an irrigation system must perform to implement an irrigation application. Usefull for irrigation_advisor services that integrate info related to irrigation systems capacity. Default is equal to 1"
+          }
+       },
+       "description": "A recommedation for an irrigation application that could be applied within a time interval of 1 or more days without adverse effects for the crop",
+       "required": [
+         "earliest",
+         "latest"
+         ]
+      },
+      "sub-Daily": {
+        'type': "object",
+        "properties": {
+          "time": {
+            "type": "string",
+            "format": "date-time",
+            "description": "the time that an irrigation implementation task should be implemented."
           }
         },
-        "example": {
-          "date": "2021-07-31",
-          "applications": [
-            {
-              "date_time": "2021-07-31T10:00:00Z"
-            },
-            {
-              "date_time": "2021-07-31T18:00:00Z"
-            }
+        "description": "A recommedation for an irrigation application that is time intensive and usually a delay of dome hours may have adverse effects on crop",
+        "required": [
+          "time"
           ]
-        }
+      },
+      "Ratio": {
+        "type": "integer",
+        "description": "The percentage of the remaining daily requirements."
```

Comment:
Should be "the percentage of the daily requirements to apply" (NOT remaining)

Reply:
I have ignorred this comment